graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -477,6 +481,17 @@ class RenameSchemaTypesVisitor(Visitor):
             "UnionTypeDefinitionNode",
         }
     )
+    name_conflicts: Dict[str, Set[str]]  # Collects naming conflict errors. If renaming
+    # would result in multiple types being renamed to "Foo", name_conflicts will map "Foo" to an
+    # set containing the name of each such type.
+    renamed_to_scalar_conflicts: Dict[str, str]  # Collects attempts to rename a type with a name
+    # already used by a scalar. If renamings tries to rename a type named "Foo" to "Bar" and there
+    # already exists a scalar named "Bar" in the schema, then
+    # renamed_to_scalar_conflicts["Foo"] == "Bar"
+    reverse_name_map: Dict[str, str]  # From renamed type name to original type name
+    # reverse_name_map contains all non-suppressed types, including those that were unchanged. Must
+    # contain unchanged names to prevent renaming conflicts and raise SchemaNameConflictError when
+    # they arise.
```

Comment:
These comments feel kind of awkward to read to me. Consider reformatting in a style like this:
```
# comment
# comment
attr: type

# comment
# comment
attr2: type
```